Modules/Core/Common/include/itkMath.h

```diff
@@ -40,6 +40,11 @@
  */
 #include <vxl_version.h>
 
+#if !defined(ITK_LEGACY_REMOVE)
+// Supress MSVC warnings
+#  define _SILENCE_NONFLOATING_COMPLEX_DEPRECATION_WARNING
```

Comment:
I think this should go into the cxx file, not the header file. Somewhere around here:
https://github.com/InsightSoftwareConsortium/ITK/blob/ea231d600298a16396a31742b5f26675e6c6334c/Modules/Core/Common/test/itkMathTest.cxx#L674